persistence/src/main/java/org/cbioportal/genome_nexus/util/PdbHeaderParser.java

```diff
@@ -1,38 +1,32 @@
-package org.cbioportal.genome_nexus.service.internal;
+package org.cbioportal.genome_nexus.util;
 
 import org.cbioportal.genome_nexus.model.PdbHeader;
-import org.cbioportal.genome_nexus.service.exception.ResourceMappingException;
 import org.springframework.stereotype.Component;
 
 import java.util.*;
 
 /**
- * Class designed to parse raw text files returned from PDB data service,
- * and to convert data into JSON objects.
+ * Class designed to parse raw PDB file text content.
  *
  * @author Selcuk Onur Sumer
  */
 @Component
-public class PdbFileParser
+public class PdbHeaderParser
```

Comment:
This needs to be moved from service to persistence layer, since now we have to do the manual parsing & mapping in the `PdbHeaderRepositoryImpl`